services/recipesServices.js

```diff
@@ -56,29 +56,37 @@ export const recipeDetails = (filters = {}) => {
   return include;
 };
 
+const validatePagination = (page, limit, maxLimit = 50) => {
```

Comment:
It would be nice to do Joi Pagination validation;
example
export const paginationSchema = Joi.object({
  page: Joi.number().integer().min(1),
  limit: Joi.number().integer().min(1),
  sort: Joi.string().valid("ASC", "DESC", "asc", "desc"),
});

export const getAllContactsSchema = paginationSchema.append({
  favorite: Joi.boolean().optional(),
});

 const { error } = schema.validate(req["query"], { abortEarly: false });
    if (error) {
      next(HttpError(400, error.message));
    }